src/socketx.h

```diff
@@ -15,5 +16,6 @@ typedef int socklen_t;
 typedef int SOCKET;
 const int INVALID_SOCKET = -1;
 inline void closesocket(int x) { close(x); }
+#define socketerror   errno
```

Comment:
Use `#define socketerror()` so the use of this doesn't look like a variable.

Reply:
Fixed.

Reply:
Also this has not been fixed.